WD-Firmware/rtos/rtos_core/freertos/config/hifive-1/FreeRTOSConfig.h

```diff
@@ -0,0 +1,168 @@
+/*
+    FreeRTOS V9.0.0 - Copyright (C) 2016 Real Time Engineers Ltd.

```

Comment:
1. use origanl (not Ronen) as base line
2. merge Ronen into it + double check Natis changes
3. configCPU_CLOCK_HZ to be in comment or take define from build system